.github/workflows/create-tag.yml

```diff
@@ -0,0 +1,27 @@
+name: Create Tag on Each Date if Any Commit
+
+on:
+  schedule:
+    # Run daily
+    - cron: '0 0 * * *'
```

Comment:
Could you choose an hour which is somewhat more "unusual"? I can imaging that many cron jobs on servers are already running at midnight. As Snap update automation is running at 7:09am in most cases, I suggest to run this job at 6:29am, shortly before Snap automation kicks in and also not on such a "usual" time.